tests/fixtures/marc/marc_record_attribute_and_subfield_variations.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let me know if this is overkill for the subfields in this fixture since there isn't a change in behavior for particular subfields

Reply:
It's probably good to include them all just for thorough testing.